ecs/task_definition_TESTNET.json

```diff
@@ -0,0 +1,186 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "fee-oracle-container-TESTNET",
+      "essential": true,
+      "image": "852551629426.dkr.ecr.us-east-2.amazonaws.com/fee-oracle:stable",
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/fee-oracle-TESTNET",
+          "awslogs-region": "us-east-2",
+          "awslogs-stream-prefix": "ecs"
+        }
+      },
+      "portMappings": [
+        {
+          "containerPort": 8091,
+          "hostPort": 8091,
+          "protocol": "tcp"
+        }
+      ],
+      "environment": [
+        {
+          "name": "DD_SERVICE",
+          "value": "fee-oracle"
+        },
+        {
+          "name": "APP_MODE",
+          "value": "release"
+        },
+        {
+          "name": "WORKING_ENV",
+          "value": "dev"
+        },
+        {
+          "name": "LOG_LEVEL",
+          "value": "5"
+        },
+        {
+          "name": "HTTP_SERVER_MODE",
+          "value": "debug"
+        },
+        {
+          "name": "HTTP_SERVER_PORT",
+          "value": "8091"
+        },
+        {
+          "name": "CONVERSION_RATE_JOB_FREQUENCY",
+          "value": "*/5 * * * *"
+        },
+        {
+          "name": "GAS_PRICE_JOB_FREQUENCY",
+          "value": "* * * * *"
+        },
+        {
+          "name": "DATA_VALID_INTERVAL",
+          "value": "3600"
+        },
+        {
+          "name": "REMOTE_PARAM_OPERATOR_ENABLE",
+          "value": "true"
+        },
+        {
+          "name": "IDENTITY_KEY_TYPE",
+          "value": "secp256k1"
+        },
+        {
+          "name": "CONVERSION_RATE_PAIRS",
+          "value": "eth,usdt,eth,matic,matic,usdt,eth,glmr,glmr,usdt"
+        }
+      ],
+      "secrets": [
+        {
+          "name": "IDENTITY_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/IDENTITY_KEY"
+        },
+        {
+          "name": "ETHERSCAN_API_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/ETHERSCAN_API_KEY"
+        },
+        {
+          "name": "POLYGONSCAN_API_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/POLYGONSCAN_API_KEY"
+        },
+        {
+          "name": "COINMARKETCAP_API_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/COINMARKETCAP_API_KEY"
+        },
+        {
+          "name": "MOONSCAN_API_KEY",
+          "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/fee-oracle/MOONSCAN_API_KEY"
```

Comment:
after you created secrets in SSM for TESTNET, replace those values in `valueFrom` and then I think fee oracle is good to go